config/locales/en/errors.yml

```diff
@@ -966,15 +966,16 @@ en:
               blank: Enter how much is left to pay on the mortgage
               not_a_number: How much there is left to pay on the mortgage must be a number, like 50
             percentage_applicant_owned:
-              invalid: Percentages entered need to total 100%
+              invalid: Percentages entered need to total 100% - check percentage owned by partner
```

Comment:
This should say `by client` instead of partner, double check the others in this file as there are a couple of mixups